wait-for-it.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: wait-for-it
+  version: 0.1.0
+  epoch: 0
+  description: Pure bash script to test and wait on the availability of a TCP host and port
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: MIT
+environment:
+  contents:
+    packages:
+      - busybox
+      - bash
+      - ca-certificates-bundle
+      - build-base
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh
```

Comment:
This should be pinned to a tagged version rather than `master` (and `package.version` should match the tagged version).  If you are taking a snapshot from git, you should specify the commit hash instead of `master`.

Reply:
_off topic and nit: just add this concern as a check item in the [linter tracking issue](https://github.com/wolfi-dev/os/issues/86), so we don't forget!_

Reply:
I've updated the version information according to your feedbacks @kaniini, thx for your help